sdcm/cluster.py

```diff
@@ -1883,25 +1885,33 @@ def install_scylla(self, scylla_repo):
-                self.remoter.run(
-                    "echo 'Acquire::Check-Valid-Until \"false\";' |sudo tee /etc/apt/apt.conf.d/99jessie-backports")
-                self.remoter.run('sudo apt-get update')
-                self.remoter.run('sudo apt-get install gnupg-curl -y')
-                self.remoter.run(
-                    'sudo apt-key adv --fetch-keys https://download.opensuse.org/repositories/home:/scylladb:/scylla-3rdparty-jessie/Debian_8.0/Release.key')
-                self.remoter.run('sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 17723034C56D4B19')
-                self.remoter.run(
-                    'echo "deb http://download.opensuse.org/repositories/home:/scylladb:/scylla-3rdparty-jessie/Debian_8.0/ /" |sudo tee /etc/apt/sources.list.d/scylla-3rdparty.list')
-                self.remoter.run('sudo apt-get update')
-                self.remoter.run('sudo apt-get install -y openjdk-8-jre-headless -t jessie-backports')
-                self.remoter.run('sudo update-java-alternatives -s java-1.8.0-openjdk-amd64')
+                install_debian_8_prereqs = dedent(r'''
+                    sed -i -e 's/jessie-updates/stable-updates/g' /etc/apt/sources.list
+                    echo "deb http://archive.debian.org/debian jessie-backports main" | sudo tee /etc/apt/sources.list.d/backports.list
+                    sed -i -e 's/:\/\/.*\/debian jessie-backports /:\/\/archive.debian.org\/debian jessie-backports /g' /etc/apt/sources.list.d/*.list
+                    echo 'Acquire::Check-Valid-Until "false";' |sudo tee /etc/apt/apt.conf.d/99jessie-backports
+                ''')
+                self.remoter.run(f'sudo bash -cxe "{install_debian_8_prereqs}"')
+                self.remoter.run("sudo apt-get update", ignore_status=True)
+                install_debian_8_prereqs = dedent('''
+                    export DEBIAN_FRONTEND=noninteractive
+                    apt-get install gnupg-curl -y
+                    apt-key adv --fetch-keys https://download.opensuse.org/repositories/home:/scylladb:/scylla-3rdparty-jessie/Debian_8.0/Release.key
+                    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 17723034C56D4B19
+                    echo "deb http://download.opensuse.org/repositories/home:/scylladb:/scylla-3rdparty-jessie/Debian_8.0/ /" |sudo tee /etc/apt/sources.list.d/scylla-3rdparty.list
+                ''')
+                self.remoter.run(f'sudo bash -cxe "{install_debian_8_prereqs}"')
+                self.remoter.run("sudo apt-get update", ignore_status=True)
+                install_debian_8_prereqs = dedent('''
+                    export DEBIAN_FRONTEND=noninteractive
+                    apt-get install -y openjdk-8-jre-headless -t jessie-backports
+                    update-java-alternatives -s java-1.8.0-openjdk-amd64
+                ''')
+                self.remoter.run(f'sudo bash -cxe "{install_debian_8_prereqs}"')
```

Comment:
Is there a reason that you separated the `install_debian_8_prereqs` to three different executions?

Reply:
i will revert it, as the manager doesn't support Debian 8 anyway.

Reply:
reverted my changes, as it is related to scylla installation... the part related to the manager, i removed from the code, as not supported.